readme.md

```diff
@@ -42,14 +46,41 @@ Either:
 - Path to an app
 - Path to a file
 
-### size
+### options
+
+Type: `object`
+
+#### size
 
 Type: `number`<br>
 Default: `1024`<br>
 Maximum: `1024`
 
 Size of the returned icon.
 
+### fileIcon.file(input, [options])
+
+Returns a `Promise` for when the file is written to `destination`.
+
+### options
+
+Type: `object`
```

Comment:
Yes that line + the destination option. But I'm fine if you are with the current docs :)